app/obfuscators/caesar_cipher.py

```diff
@@ -0,0 +1,47 @@
+from random import randint
+
+from app.utility.base_obfuscator import BaseObfuscator
+
+
+class Obfuscation(BaseObfuscator):
+
+    @property
+    def supported_platforms(self):
+        return dict(
+            windows=['psh'],
+            darwin=['sh'],
+            linux=['sh']
+        )
+
+    """ EXECUTORS """
+
+    def psh(self, link):
+        decrypted = self.decode_bytes(link.command)
+        encrypted, shift = self._apply_cipher(decrypted)
+        link.pin = shift
```

Comment:
why link.pin = shift?

Reply:
I was using the value to potentially write out a different test case for the caesar cipher algorithm. This will not be needed anymore for the current tests, so I have removed it.